src/CommandLoader.php

```diff
@@ -36,6 +37,23 @@ public function get(string $name)
             throw new CommandNotFoundException(sprintf('Command "%s" does not exist.', $name));
         }
 
+        /** @var Command $commandClass */
+        $commandClass = $this->commandMap[$name];
+        $description = $commandClass::getDefaultDescription() ?? $this->getCommandInstance($name)->getName() ?? '';
+
+        return new LazyCommand(
+            $name,
+            [],
```

Comment:
What about aliases? In Symfony they stored in default name with `|` separator:
```
protected static $defaultName = 'app:my-command|project:my-command|legacy:foo';
```
It may be useful while refactoring

Reply:
Added that and hidden commands. It doesn't work too well though.